gradle/artifact-settings.gradle

```diff
@@ -0,0 +1,14 @@
+ext {
+    navigationArtifactGroupId = 'com.mapbox.mapboxsdk'
```

Comment:
So we can't change the artifact group id?

Reply:
This was a test, but has not affected anything. Jitpack is overwriting this by the Github repo names. But change this to *MapLibre* will part of the `v11` PR.